game/gamestates/animation.lua

```diff
@@ -59,8 +59,6 @@ function state:update(dt)
     if not _animation_task or _animation_task:done() then
       SWITCHER.pop(_alert)
     end
-  else
-    _view.sector:updateVFX(dt)
```

Comment:
Is it ok to remove this line? Can't test right now, but will later.

Reply:
yes, this function does absolutely nothing. i even removed it